examples/guestbook-go/BUILD

```diff
@@ -34,6 +34,9 @@ filegroup(
 
 filegroup(
     name = "all-srcs",
-    srcs = [":package-srcs"],
+    srcs = [
+        ":package-srcs",
+        "//examples/guestbook-go/_src:all-srcs",
```

Comment:
This directory doesn't exist on master. What creates it on your dirty workspace?